sdk/include/ndk/rtlfuncs.h

```diff
@@ -3979,9 +3979,9 @@ NTSYSAPI
 ULONG
 NTAPI
 LdrRelocateImageWithBias(
```

Comment:
Committed.

Reply:
OK this was the suggestion that currently breaks compilation. Please revert that change you did so that we can merge yoru PR asap. I'll implement the suggestion separately later.

Reply:
Committed.